### PR TITLE
Remove depreciated Google Analytics CSP

### PR DESCRIPTION
**Pull request by llienher**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml [hunk 1]
```diff
@@ -950,8 +950,7 @@ vars:
   content_security_policy_main_default_src_extra: ''
   content_security_policy_main_script_src_extra: '
     https://maps.google.com/
-    https://maps.googleapis.com/
-    https://www.google-analytics.com/
+    https://*.googleapis.com/
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:953]: Why not keep the www?

Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:953]: Just copying what they had for Google Maps CSP, they apparently not put "map", that is not www that I removed :D

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:953]: Here: https://developers.google.com/tag-platform/tag-manager/csp?hl=fr#google_analytics_4_google_analytics?
There is no googleapis.com on script-src ...
Can you also add the previous URL in comment?

Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:953]: You are confusing Analytics and Maps, both have their respective URLs: https://developers.google.com/maps/documentation/javascript/content-security-policy?hl=fr#allowlist_csp

And actually even maps.google should become *.google so I need to tweek thas as well.

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:953]: +1 Then one more reason to put the links as comments :-)

Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:955]: Ok, I did add comments, is it okay for comprehension ?

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml [hunk 2]
```diff
@@ -948,10 +948,11 @@ vars:
   lingua_extractor: {}
 
   content_security_policy_main_default_src_extra: ''
+  # Google Maps uses map.google.com and map.googleapis.com
+  # Google Analytics 4 uses googletagmanager.com
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml:952]: I think that the links are also a good ressource =>
```suggestion
  # Google Maps uses map.google.com and map.googleapis.com
  # More info: https://developers.google.com/maps/documentation/javascript/content-security-policy?hl=fr#allowlist_csp
  # Google Analytics 4 uses googletagmanager.com
  # More info: https://developers.google.com/tag-platform/tag-manager/csp?hl=fr#google_analytics_4_google_analytics
```

Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml]: Ok, done.

**Review by sbrunner — APPROVED** [review]
Thanks :-)

**Pull request merged**